operations/helm/charts/mimir-distributed/templates/validate.yaml

```
@@ -143,3 +143,7 @@
     {{- fail "memcached-results.enabled has been renamed to results-cache.enabled" }}
   {{- end }}
 {{- end }}
+
+{{- if not .Values.ingester.statefulSet.enabled -}}
+{{- fail "Ingesters are stateful, kubernetes Deployment workloads are no longer supported. Please refer to the documentation for more details." }}
```

```suggestion
{{- fail "You have set ingester.statefulSet.enabled: false. Ingesters are stateful, kubernetes Deployment workloads are no longer supported. Please refer to the documentation for more details." }}
```

do we any actual documentation about this? If not consider removing the last sentence